app/templates/views/accessibility_statement.html

```diff
@@ -166,11 +158,11 @@ <h3 class="heading-small" id="pdfs-documents">
     We provide a written description of the information in the PDF on an HTML page as an accessible alternative.
   </p>
 
-  <h3 class="heading-small" id="emails">
+ <!-- <h3 class="heading-small" id="emails">
     Emails
   </h3>
 
-  <p class="govuk-body">
+  <<p class="govuk-body">
```

Comment:
[issue (nonblocking)] accidental double `<` 

```suggestion
  <p class="govuk-body">
```